content/news/events-february-2025.md

```diff
@@ -0,0 +1,61 @@
++++
+headline = "ARTICLE"
+title = "Foulab Events - February 2025"
+lastmod = "2025-01-30 21:52:33-0400"
+++++
```

Comment:
I think Hugo frontmatter ends with 3 plusses: https://gohugo.io/content-management/front-matter/

You had it right at first.